events/HF_battle_events.txt

```diff
@@ -2529,38 +2555,91 @@ character_event = {
 		}
 		add_trait = desert_terrain_leader
 	}
+
```

Comment:
Starting from here. Why it's commented out? These are the same options. Only triggers should be modified.